unconstrained_test.go

```diff
@@ -1503,16 +623,5 @@ func testMinimize(t *testing.T, tests []UnconstrainedTest, method Method) {
 				}
 			}
 		}
-
```

Comment:
Why is this difficult to pass? Shouldn't it just exit out at the first iteration because the location is already optimal?